corehq/pillows/mappings/user_mapping.py

```diff
@@ -95,7 +96,20 @@
                                        },
                                        'type': 'object'},
                 'status': {'type': 'string'},
-                'user_data': {'type': 'object', 'enabled': False},
+                'user_data_es': {
+                    'type': 'nested',
+                    'dynamic': False,
+                    'properties': {
+                        'key': {
+                            'type': 'string',
+                            'index': 'not_analyzed',
+                        },
+                        'value': {
+                            'type': 'string',
+                            'index': 'not_analyzed',
```

Comment:
yeah i'm not sure either to be honest. kind of an arbitrary decision based on the requirement that we didn't need fuzzy searching. can make it analyzed